cluster/config-defaults.yaml

```diff
@@ -606,6 +606,7 @@ kubelet_image_gc_low_threshold: 40
 {{if eq .Cluster.Environment "production"}}
 teapot_admission_controller_validate_application_label: "true"
 teapot_admission_controller_validate_base_images: "true"
+teapot_admission_controller_validate_base_images_namespaces: "^kube-system$"
```

Comment:
Doesn't this enable the base image check _only_ in `kube-system` this way?

![image](https://github.com/user-attachments/assets/ee96a05c-6780-4b79-88ba-3925be4ebb13)

https://github.bus.zalan.do/teapot/admission-controller/blob/f3b5344ce3ad2efdf7fee04c484a8311f9263e57/pkg/config/config.go#L88-L89

Reply:
Yeah, I realized this is at least wrong.

Will try to replicate the prod issue in a pet cluster to better think about a solution.